eng/Tools.props

```diff
@@ -80,7 +80,7 @@
   <Target Name="InitGlobalTools" Condition="'@(RepoTool)' != ''" AfterTargets="Restore">
 
     <PropertyGroup>
-      <GlobalToolsDir>$(ArtifactsDir)tools</GlobalToolsDir>
+      <GlobalToolsDir>$([MSBuild]::NormalizePath('$(RepoRoot)', '.tools', 'globaltools'))</GlobalToolsDir>
```

Comment:
Is a comment worth here pointing out we should keep on sync with Directory.Build.props? Probably won't change too often though.

Reply:
good idea. thanks.